app/assets/stylesheets/application.scss

```diff
@@ -1,3 +1,6 @@
+@import url('https://fonts.googleapis.com/css2?family=Are+You+Serious&family=Audiowide&family=Cinzel+Decorative&family=Cinzel:wght@500&family=Libre+Barcode+128+Text&family=Comforter+Brush&family=Cormorant+Infant:ital,wght@1,300&family=Cutive+Mono&family=Explora&family=Great+Vibes&family=IM+Fell+English:ital@1&family=Libre+Barcode+39+Text&family=Monofett&family=New+Rocker&family=Niconne&family=Orbitron&family=Passions+Conflict&family=Philosopher&family=Playball&family=Rock+Salt&family=Sacramento&family=Special+Elite&family=Stardos+Stencil&family=Yuji+Syuku&display=swap');
```

Comment:
that's a lot of fonts! In a production project this would be a big no no. If you truly need all these fonts you should add them as a local asset. There are ways to optimize this. Just an FYI...